src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx

```diff
@@ -20,7 +20,15 @@ import store from '@/store/index';
 import { DATA_TABLE_EVENTS } from '@/constants/events';
 import { detectMajorChiVersion } from '@/utils/utils';
 
-Vue.config.ignoredElements = ['chi-popover'];
+Vue.config.ignoredElements = [
+  'chi-date-picker',
+  'chi-popover',
+  'chi-text-input',
+  'chi-label',
+  'chi-button',
+  'chi-time-picker',
+  'chi-number-input',
```

Comment:
These are the ones you currently use in the `DataTableClient.vue` example. What if the developers use for example `chi-spinner`, `chi-textarea`, etc ? I was planning yo make a separate task to investigate how to do `ignoreElements` like `chi-*` globally on Chi Vue level. Please, look into it if possible.

Reply:
Sure! Should i keep these in the meantime?